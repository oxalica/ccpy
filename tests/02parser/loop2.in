while 1, 2:
  pass
