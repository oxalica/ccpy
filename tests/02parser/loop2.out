Fail: Expect `:` after `while` condition
